### PR TITLE
Add pink color for PinetimeStyle Watchface

### PR DESCRIPTION
**Pull request by Akselmo**
Let's try this again. This was previously attempted at https://github.com/InfiniTimeOrg/InfiniTime/pull/1300

I had some trouble with accidentally modifying the *submodules* instead of the InfiniTime project itself, so the tests kept failing.
Oh well, live and learn!

image

Anyhow, here it is, a pretty pink color!

As per the previous PR:

My fiancee was absolutely devastated there was no proper pink color in the device, so I wanted to add it.

Apparently this is the true pink color. :smile: 

She was happy with it, I hope you are too. :smile:

### CODE REVIEW

File: src/components/settings/Settings.h [hunk 1]
```diff
@@ -245,4 +246,4 @@ namespace Pinetime {
       void SaveSettingsToFile();
     };
   }
-}
+}
```
Comment (NeroBurner) [inline src/components/settings/Settings.h:249]: Please don't remove the last empty line at the end of the files. Try to keep the commits clean.

```suggestion
}

```

File: src/displayapp/Colors.cpp [hunk 2]
```diff
@@ -39,7 +39,9 @@ lv_color_t Pinetime::Applications::Convert(Pinetime::Controllers::Settings::Colo
       return LV_COLOR_MAKE(0xb0, 0x0, 0xb0);
     case Pinetime::Controllers::Settings::Colors::Orange:
       return LV_COLOR_ORANGE;
+    case Pinetime::Controllers::Settings::Colors::Pink:
+      return LV_COLOR_MAKE(0xFF, 0xAE, 0xC9);
     default:
       return LV_COLOR_WHITE;
   }
-}
+}
```
Comment (NeroBurner) [inline src/displayapp/Colors.cpp:47]: Please don't remove the last empty line at the end of the files. Try to keep the commits clean.

```suggestion
}

```

File: src/displayapp/screens/WatchFacePineTimeStyle.cpp [hunk 3]
```diff
@@ -585,7 +585,7 @@ Pinetime::Controllers::Settings::Colors WatchFacePineTimeStyle::GetPrevious(Pine
   if (colorAsInt > 0) {
     prevColor = static_cast<Controllers::Settings::Colors>(colorAsInt - 1);
   } else {
-    prevColor = static_cast<Controllers::Settings::Colors>(16);
+    prevColor = static_cast<Controllers::Settings::Colors>(17);
   }
   return prevColor;
-}
+}
```
Comment (NeroBurner) [inline src/displayapp/screens/WatchFacePineTimeStyle.cpp:591]: Please don't remove the last empty line at the end of the files. Try to keep the commits clean.

```suggestion
}

```

**Review by NeroBurner — CHANGES_REQUESTED** [review]
Other than the removed trailing newlines looking good :purple_heart:

**Review by NeroBurner — APPROVED** [review]
LGTM

**Comment by JF002** [comment]
@Akselmo Thank you for your first contribution in InfiniTime :+1: and sorry for the time it took to review it. I hope your fiancee is still happy with her PineTime!

**Comment by Akselmo** [comment]
> @Akselmo Thank you for your first contribution in InfiniTime +1 and sorry for the time it took to review it. I hope your fiancee is still happy with her PineTime!

Thanks and no problem. She likes it even better now with the pink color! :smile:

**Pull request merged**